app/containers/LanguageProvider/tests/index.test.js

```diff
@@ -28,17 +27,15 @@ describe('<LanguageProvider />', () => {
     const children = <h1>{text}</h1>;
     const { queryByText } = render(
       <Provider store={store}>
-        <LanguageProvider messages={messages} locale="en">
-          {children}
-        </LanguageProvider>
+        <LanguageProvider messages={messages}>{children}</LanguageProvider>
       </Provider>,
     );
     expect(queryByText(text)).toBeInTheDocument();
   });
 
   it('should render the default language messages', () => {
     const { queryByText } = render(
-      <Provider store={store}>
+      <Provider store={configureStore({ language: { locale: undefined } })}>
```

Comment:
Because `DEFAULT_LOCALE` is set to `en` in `app/locales.js`, we can't properly test the `defaultMessage` value unless we set the language store locale to be `undefined`.